src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js

```diff
@@ -1243,6 +1243,19 @@ describe('ReactDOMServerIntegration', () => {
         );
       });
 
+      itRenders('svg element with a tabIndex attribute', async render => {
+        let e = await render(<svg tabIndex="1" />);
+        expect(e.tabIndex).toBe(1);
+      });
+
+      itRenders(
+        'svg element with a badly cased tabIndex attribute',
+        async render => {
+          let e = await render(<svg tabindex="1" />, 1);
+          expect(e.tabIndex).toBe(1);
+        },
+      );
```

Comment:
Why is this expected to work if SVG is case sensitive?

Reply:
It might differ in another browser, but a quick check in Chrome shows that it works, and it gets assigned as `tabindex`:

https://codepen.io/anon/pen/PJeoeE

<img width="552" alt="screen shot 2017-10-09 at 2 51 16 pm" src="https://user-images.githubusercontent.com/590904/31353666-61fdbd14-ad01-11e7-9b2e-05fcf01c5729.png">